e2e_samples/dataset_versioning/insert_sql/README.md

```diff
@@ -0,0 +1,29 @@
+# How to setup
+
+## Install Libraries
+### Devcontainer
+If you are using devcontainer, you can just open in devcontainer and everything required should be installed .
+
+### Local
+If you are not using devcontainer, you need to install required lib with ```pip install requirements.txt```
+
+## Login to azure
+```az login```
+
+Now you are good to run python main.py with different flags. You can get more details with -h flag.
+
```

Comment:
maybe you can add one line here just to show what to pass into the script?
( i know you have the -h flag, just thinking it may still be good to include a example here :)

Reply:
Added